### PR TITLE
[ETFE-3162] - Add fixed transport installation journey type page content + align logic

### PR DESCRIPTION
**Pull request by LewisMitchell**
~Need to cleanup some imports, run the JT's (will probably break) and PT's~

### CODE REVIEW

File: app/viewmodels/checkAnswers/sections/journeyType/HowMovementTransportedSummary.scala [hunk 1]
```diff
@@ -50,7 +52,13 @@ object HowMovementTransportedSummary {
               ).url,
               id = HowMovementTransportedPage
             ).withVisuallyHiddenText(messages("howMovementTransported.change.hidden"))
-          )} else Seq()
+          )
         )
     }
+
+  private def showChangeLink(implicit request: DataRequest[_]): Boolean =
```
Comment (machinesleet) [inline app/viewmodels/checkAnswers/sections/journeyType/HowMovementTransportedSummary.scala:59]: Would it be clearer to remove the negation in this method, and at the call site, and just call this something like `hideChangeLink` instead?

File: app/controllers/sections/transportUnit/TransportUnitIndexController.scala
Comment (machinesleet) [inline app/controllers/sections/transportUnit/TransportUnitIndexController.scala:1]: as discussed, could the `saveAnswersAndRedirect` method not use currying

File: app/viewmodels/checkAnswers/sections/journeyType/HowMovementTransportedSummary.scala (hunk 1 above)
Comment (LewisMitchell) [reply app/viewmodels/checkAnswers/sections/journeyType/HowMovementTransportedSummary.scala]: Done in [bdaf620](https://github.com/hmrc/emcs-tfe-change-destination-frontend/pull/30/commits/bdaf62069222611d2f5a35a5907fc3aad754e370)

File: app/controllers/sections/transportUnit/TransportUnitIndexController.scala
Comment (LewisMitchell) [reply app/controllers/sections/transportUnit/TransportUnitIndexController.scala:1]: Done

**Comment by LewisMitchell** [comment]
I need to check the logic again (post-rebase) on Monday and run it past UCD, going to mark it as a draft

**Review by machinesleet — APPROVED** [review]

File: test/views/sections/journeyType/HowMovementTransportedNoOptionViewSpec.scala [hunk 2]
```diff
@@ -14,7 +14,6 @@
  * limitations under the License.
  */
 
-// wee
```
Comment (machinesleet) [inline test/views/sections/journeyType/HowMovementTransportedNoOptionViewSpec.scala:17]: can never be too careful

**Review by machinesleet — APPROVED** [review]

**Review by KidiIT — COMMENTED** [review]
All reviewed and approved🙌❤️

**Comment by KidiIT** [comment]
All checked and approved🫶❤️

**Comment by KidiIT** [comment]
All changes has been staged. Thank you and you're welcome😉❤️

**Pull request merged**